ui/karma.conf.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(config) {
       'spec/spec-helper.js': ['webpack', 'sourcemap'],
       'spec/index.js': ['webpack', 'sourcemap'],
     },
-    reporters: ['dots'],
+    reporters: ['dots'], // can also use 'verbose' and/or 'progress' for more detailed reporting
```

Comment:
I've tried --reporters=verbose but can't get it to work. That would be preferable.